### ISSUE TITLE
run on custom dataset

### ISSUE THREAD

**Issue by deepai-lab** [beric7/corrosion_cs_classification#3]
Hi @beric7,

I am trying to run the code on my customized dataset and I prepared the dataset using the description you provided. I have 12 classes. Now I am getting an error. Should I have to change the color mapping for 12 classes as this dataset has 4 classes in datahandler_plus.py file? If so, how can I get the exact color code?

**Comment by beric7** [comment]
@deepai-lab yes you will need to update the color mapping code for the number of classes you are using. To get that exact color code, there are a number of ways you could do this, however, I do not think I have a script for this on this repo. I could try and find the script I used to do this, or you could try and write a customizable one yourself. I would suggest converting the RGB image into a list. Such that if the image was 100x100 that you would have 10,000 positions with (R, G, B). Then turn that list into a set, and your 12 colors will remain. It will be up to you to figure out how to then map those colors to your classes, but at least you will have the exact colors.

**Comment by Wang2207** [comment]
I can't download the dataset, can you upload the dataset？

**Comment by beric7** [comment]
@Wang2207 - Can you confirm that which DOI you followed for downloading the dataset? Are you downloading the correct version?

**Comment by beric7** [comment]
@deepai-lab Closing this issue in 2 weeks if I do not hear back from you. I hope it worked out!
